packages/react-examples/src/react/TextField/TextField.ErrorMessage.Example.tsx

```diff
@@ -19,6 +21,17 @@ const getErrorMessagePromise = (value: string): Promise<string> => {
   });
 };
 
+const getRichErrorMessage = (value: string) => {
+  return value.length < 3 ? (
+    ''
+  ) : (
+    <Stack styles={{ root: { height: 24 } }} verticalAlign="center" horizontal tokens={{ childrenGap: 8 }}>
```

Comment:
A couple minor issues here: typically in our examples we try to avoid creating objects inline, since in real code that can cause major performance issues (so we don't want our examples to encourage that practice). Easiest thing to do here would be putting all the stack props in a file-level const. `const stackProps: IStackProps = { /* all the prop values here */ }`

Similar for the icon styles a couple lines down.

Reply:
done.